web/src/components/planner/Statistics.vue

```diff
@@ -0,0 +1,327 @@
+          <h1 class="text-h5 mb-4">
+            <i class="fas fa-globe" />
+            <span class="ml-3">Raw Resources</span>
+          </h1>
+          <p v-show="helpText" class="mb-4">
+            <i class="fas fa-info-circle" /> Shows the amount of raw resources
+            consumed by all your factories.
+          </p>
+          <div v-if="allFactoryRawResources.length > 0">
+            <span v-for="(resource, id) in allFactoryRawResources" :key="id">
+              <v-chip class="sf-chip blue" variant="tonal">
+                <game-asset :subject="resource.id.toString()" type="item" />
+                <span class="ml-2">
+                  <b>{{ getPartDisplayName(resource.id.toString()) }}</b>: {{ formatNumber(resource.totalAmount) }}/min
+                </span>
+              </v-chip>
+            </span>
+          </div>
+          <p v-else class="text-body-1">Awaiting Resource Consumption</p>
+          <v-divider class="my-4 mx-n4" color="white" thickness="5px" />
+
+          <!-- Building Summary Area -->
+          <h1 class="text-h5 mb-4">
+            <i class="fas fa-building" />
+            <span class="ml-3">Building Summary</span>
+          </h1>
+          <p v-show="helpText" class="mb-4">
+            <i class="fas fa-info-circle" /> Shows the amount buildings of each
+            type in all your factories.
+          </p>
+          <div v-if="totalBuildingsByType.length > 0">
+            <span v-for="(building, type) in totalBuildingsByType" :key="type">
+              <v-chip class="sf-chip orange" variant="tonal">
+                <game-asset :subject="building.name" type="building" />
+                <span class="ml-1">
+                  <b>{{ getBuildingDisplayName(building.name) ?? "UNKNOWN" }}</b>: {{ formatNumber(building.totalAmount) ?? 0 }}x
+                </span>
+              </v-chip>
+            </span>
+          </div>
+          <p v-else class="text-body-1">Awaiting Building Construction</p>
+
+          <v-divider class="my-4 mx-n4" color="white" thickness="5px" />
+
+          <!-- Excess Product Area -->
+          <h1 class="text-h5 mb-4">
+            <i class="fas fa-warehouse" />
+            <span class="ml-3">Product Surplus & Deficit</span>
+          </h1>
+          <p v-show="helpText" class="mb-4">
+            <i class="fas fa-info-circle" /> Shows the amount of surplus or
+            deficit of items you have in your factory. These are items that
+            either need to be produced more (in red), or items that can be
+            stored or sunk (in green)!
+          </p>
+          <div v-if="factoryProductDifferences.length > 0">
+            <v-chip
+              v-for="(product) in factoryProductDifferences"
+              :key="product.id"
+              class="sf-chip"
+              :class="{
+                'text-green': product.totalDifference > 0,
+                'text-red': product.totalDifference < 0,
+              }"
```

Comment:
```suggestion
              :class="{
                'green': product.totalDifference > 0,
                'red': product.totalDifference < 0,
              }"
```
With the `sf-chip` component I've got various helper classes that format the chips in a consistent manner. This will turn both the text green / red and apply a boarder as well.

Reply:
I'd also double check the reactivity on this, I've had lots of issues getting it to react properly when a product falls out of deficit.